NickvisionTubeConverter.WinUI/Views/MainWindow.xaml.cs

```diff
@@ -212,6 +231,6 @@ private void NotificationSent(object? sender, NotificationSentEventArgs e)
     /// <param name="e">RoutedEventArgs</param>
     private async void AddDownload(object sender, RoutedEventArgs e)
     {
-        
+
```

Comment:
empty function?

Reply:
That will be for the AddDownloadDialog when I implement it in WinUI...but that's something for #50 not this PR :)